FsAutoComplete/CommandResponse.fs

```diff
@@ -258,7 +266,7 @@ module CommandResponse =
                   [ for o in meth.Methods do
                      let tip = TipFormatter.formatTip o.Description
                      yield {
-                       Tip = tip
+                       Tip = {Signature = fst tip.Head; Comment = snd tip.Head } 
```

Comment:
Hmm... Haven't thought about 0 case. But I'm not sure if method overload can have more than one tooltip. We can make `Tip` filed in `Overload` type a list instead.